### PR TITLE
CMO-373: CANS API Changes per IV&V Findings

### PR DESCRIPTION
**Pull request by david-smith-osi**
## JIRA
https://osi-cwds.atlassian.net/browse/CMO-373

## Description
Java fixes for IV&V PDE findings.

## Tests
- [ ] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters and static analyzers for the project (SonarQube, eslint, rubocop, etc).
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.

### CODE REVIEW

File: src/test/java/gov/ca/cwds/cans/service/ClientsServiceTest.java [hunk 1]
```diff
@@ -53,7 +53,12 @@ public void getAssessmentComparison_returnsAssessmentComparisonDto() throws IOEx
             "O9kIYi80Ki", ASSESSMENT_COMPARISON_SIZE, false))
         .thenReturn(assessments);
```
Comment (tparkerosi) [inline src/test/java/gov/ca/cwds/cans/service/ClientsServiceTest.java]: Why do we have systetm outs? We should remove them

**Comment by david-smith-osi** [comment]
retest this please

File: src/test/java/gov/ca/cwds/cans/service/ClientsServiceTest.java (hunk 1 above)
Comment (david-smith-osi) [reply src/test/java/gov/ca/cwds/cans/service/ClientsServiceTest.java]: removed. i was checking whether the parameter variable changed.

**Comment by david-smith-osi** [comment]
retest this please

**Pull request merged**